test/e2e-common.sh

```diff
@@ -210,6 +210,11 @@ function install_sugar() {
 }
 
 function unleash_duck() {
+  echo "enable debug logging"
+  cat test/config/config-logging.yaml | \
+    sed "s/namespace: ${KNATIVE_DEFAULT_NAMESPACE}/namespace: ${TEST_EVENTING_NAMESPACE}/g" | \
+    ko apply --strict -f - || return $?
+
```

Comment:
you missed the `scale_controlplane mt-broker-controller` change 😞

Reply:
yeah... fixed.